jbmc/src/Makefile

```diff
@@ -48,3 +48,13 @@ $(patsubst %, %_clean, $(DIRS)):
 .PHONY: cprover_clean
 cprover_clean:
 	$(MAKE) $(MAKEARGS) -C $(CPROVER_DIR)/src clean
+
+# extended JBMC models download, for your convenience
+
+java-models-library-download:
+	@echo "Downloading java models library"
+	@curl -L https://github.com/diffblue/java-models-library/archive/master.zip > java-models-library.tar.gz
```

Comment:
This seems wrong - you are downloading a .zip file, but saving it with a .tar.gz extension...

Reply:
oops, it still works but its misleading, fixed.